spark-cassandra-connector/src/it/scala/org/apache/spark/sql/cassandra/CassandraSQLPredicatePushdownSpec.scala

```diff
@@ -50,29 +54,29 @@ class CassandraSQLPredicatePushdownSpec extends SparkCassandraITFlatSpecBase {
 
   it should "allow predicate pushdown if all partition columns have EQ predicate.  a=1 and b=1 and c=1 and d =1 are pushed down" in {
     val plan = cc.sql("select * from test_pd where a=1 and b=1 and c=1 and d=1").queryExecution.sparkPlan
-    plan.nodeName shouldBe "CassandraTableScan"
-    plan.children should have size 0
+    plan.nodeName shouldBe "Filter"
+    plan.children should have size 1
```

Comment:
manual check it the test log for now. It's hard to check it for now. I will create another ticket to better test pushdown. Idea is to only test Pushdown class by passing the whole sql query process